config/scheduler-chromeos.yaml

```diff
@@ -128,6 +128,71 @@ scheduler:
   - job: tast-basic-x86-stoneyridge
     <<: *test-job-stoneyridge
 
+  - job: tast-decoder-v4l2-sl-av1-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: &mediatek-av1-platforms
+      - mt8195-cherry-tomato-r2
+
+  - job: tast-decoder-v4l2-sl-h264-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms:
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: tast-decoder-v4l2-sl-hevc-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-av1-platforms
+
+  - job: tast-decoder-v4l2-sl-vp8-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: &mediatek-vp8-platforms
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: tast-decoder-v4l2-sl-vp9-group1-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-vp8-platforms
+
+  - job: tast-decoder-v4l2-sl-vp9-group2-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-vp8-platforms
+
+  - job: tast-decoder-v4l2-sl-vp9-group3-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-vp8-platforms
+
+  - job: tast-decoder-v4l2-sl-vp9-group4-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-vp8-platforms
+
+  - job: tast-decoder-v4l2-sf-h264-arm64-qualcomm
+    <<: *test-job-qualcomm
+
+  - job: tast-decoder-v4l2-sf-hevc-arm64-qualcomm
```

Comment:
I guess trogdor chromebooks (lazor, kingoftown) are not enabled in the new system yet, right?

Reply:
Actually both versions are enabled, and `*test-job-qualcomm` runs tests on both (and only those, as they're the only qcom-based chromebooks in the lab)

Reply:
Ah right, I missed the `qualcomm-platforms` definition in `scheduler-chromeos.yaml` - thanks for the confirmation!